macros/LIGen.mac

```diff
@@ -107,23 +107,26 @@
 # with random times between 20 and 21 ns.
 /mygen/nphotons 10000
 # Specify the injector type: collimator, diffuser or laser ball
-/mygen/injectorType collimator
+/mygen/injectorType diffuser
 # Specify the injector index for this type of LI that corresponds to the 
 # device position and profile you wish to use
-/mygen/injectorIdx 1
-#/mygen/injectorFile somefile.json
+/mygen/injectorIdx 0
+/mygen/injectorDetector ID
+/mygen/injectorFilename LightInjectors.json
+/mygen/injectorDetails LightInjectorsDetails.json
 # Default /mygen/photonMode 0 generates photon directions using the LI profile
 # Set /mygen/photonMode 1 to read pre-generated photon positions and directions 
 # from the 'photonsfile' specified in the injector file.
-#/mygen/photonMode 1
+# /mygen/photonMode 1
+
 
 ## change the name of the output root file, default = wcsim.root
 
-/WCSimIO/RootFile wcsim_LIGen.root
+/WCSimIO/RootFile wcsim_LIGen_new.root
```

Comment:
```suggestion
/WCSimIO/RootFile wcsim_LIGen.root
```
It's up to you as calibration group representive (so feel free to ignore this), but for future users the word "new" means nothing here